README.md

```diff
@@ -97,6 +97,28 @@ using different configurations. Check the `./getting-started` directory for more
 - `kubectl get deployment -n polaris` - To check the status of the deployment.
 - `kubectl describe deployment polaris-deployment -n polaris` - To troubleshoot if things aren't working as expected.
 
+#### Configuring Polaris
+
```

Comment:
Can we link this [page](https://polaris.apache.org/in-dev/unreleased/configuration/) instead of duplicating it here?

Reply:
Sure. Updated